airflow/models.py

```diff
@@ -807,6 +808,9 @@ def get_hook(self):
             elif self.conn_type == 'cassandra':
                 from airflow.contrib.hooks.cassandra_hook import CassandraHook
                 return CassandraHook(cassandra_conn_id=self.conn_id)
+            elif self.conn_type == 'cloudsql':
```

Comment:
`cloudsql` -> `gcp_ cloudsql`

Reply:
Same. No problem.

Reply:
Same here. Had to change to `gcpcloudsql`